src/components/App/App.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import DatabaseMenu from '../DatabaseMenu/DatabaseMenu';
+import IncidentsMenu from '../IncidentsMenu/IncidentsMenu';
+import './App.css';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import Profiles from '../Profiles/Profiles';
+import Passport from '../Passport/Passport';
+import ChartView from '../ChartView/ChartView';
+import IncidentInfo from '../IncidentInfo/IncidentInfo';
+
+const App = () => {
+
+  return (
+    <main className='app'>
+      <div className='content-wrap'>
+        <Header />
+        <Switch>
+          <Route exact path='/' component={IncidentsMenu} />
+          <Route path='/database' component={DatabaseMenu} />
+          <Route path='/profiles' component={Profiles} />
+          <Route path='/passport' component={Passport} />
+          {/* below is test-only for specific incident view */}
+          <Route path='/incident-details/:id' render={({ match }) => {
+            const { id } = match.params;
+            return (<IncidentInfo id={id} />)
+          }} />
+          <Route path='/incident/:id' render={({ match }) => {
+            const { id } = match.params;
+            return (<ChartView id={id} />)
+          }} />
+        </Switch>
+        <Footer />
+      </div>
+    </main>
+  )
+
+}
+
+export default App
```

Comment:
Do we still have two versions of App in components and Components folders?

Reply:
It appears that this extra Components folder may only be on your local machine.  Let's do a bit more research on that on Cameron and I's end.  This PR does not have 2 folders for components. So I'm thinking hopefully a reset on your end is a solution.

Reply:
Really weird behavior here - upon examining the remote branch on GitHub, we did discover a duplicate folder called Components containing App.js and App.css. However, that folder was not visible on our editors when we pulled the branch down locally. So we ultimately resorted to deleting the the duplicate files from the diff view in GitHub's UI. Hopefully this solves the problem